src/compiler/Parser/InkParser.ts

```diff
@@ -110,6 +115,7 @@ export class InkParser extends StringParser {
   }
 
   // Main entry point
+  // NOTE: This method is named Parse() in upstream.
```

Comment:
FYI : Naming it Parse broke the inheritance (StringParse.Parse has nothing to do with this method)